spec/helpers/medications_helper_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+describe MedicationsHelper do
+  let(:current_user) { create(:user) }
+  let(:medication) { create(:medication, user: current_user) }
+
+  before(:each) do
+    @medication = medication
+  end
+
+  def get_field(field_name)
+    @fields.find {|f| f[:id] == field_name}
```

Comment:
What's the purpose of this method. It just seems wrong to compare name with id

Reply:
This method just finds the field where the field id is `field_name`. The id in this case is the field name. [Example](https://github.com/ifmeorg/ifme/blob/master/app/helpers/medications_helper.rb#L25).